examples/oft-composer-library/layerzero.config.ts

```diff
@@ -0,0 +1,60 @@
+import { EndpointId } from '@layerzerolabs/lz-definitions'
+
+import type { OAppOmniGraphHardhat, OmniPointHardhat } from '@layerzerolabs/toolbox-hardhat'
+
+const sepoliaContract: OmniPointHardhat = {
+    eid: EndpointId.SEPOLIA_V2_TESTNET,
+    contractName: 'MyOFT',
+}
+
+const fujiContract: OmniPointHardhat = {
+    eid: EndpointId.AVALANCHE_V2_TESTNET,
```

Comment:
I would argue for reducing the mesh to only two chains. More so due to annoyance from getting testnet tokens nowadays. We need to optimize examples to being completed in the fastest time possible while still taking the dev through the main objective. The main objective of this example is the composer rather than demonstrating a mesh can have more than 2 chains in it.